src/app/components/source/UserInfo.js

```diff
@@ -20,6 +20,7 @@ const messages = defineMessages({
 
 const UserInfo = (props) => {
   if (props.user.source === null) return null;
+  const isUserSelf = (props.user.dbid === props.context.currentUser.dbid);
```

Comment:
We should do a safe property access here:

```js
const isUserSelf = (props.user?.dbid && props.user?.dbid === props.context?.currentUser?.dbid);
```

(We need the first check so that we don't end up with `undefined === undefined` giving us `true`)

Reply:
Done